ocs/ocp.py

```diff
@@ -251,3 +252,35 @@ def wait_for_resource(
                 return True
 
         return False
+
+
+def switch_to_project(project_name):
+    """
+    Switch to another project
+
+    Args:
+        project_name (str): Name of the project to be switched to
+
+    Returns:
+        bool: True on success, False otherwise
+    """
+    log.info(f'Switching to project {project_name}')
+    cmd = f'oc project {project_name}'
+    success_msgs = [
+        f'Now using project "{project_name}"',
+        f'Already on project "{project_name}"'
+    ]
+    ret = run_cmd(cmd)
+    if any(msg in ret for msg in success_msgs):
+        return True
+    return False
+
+
+def switch_to_default_project():
+    """
+    Switch to default project
+
+    Returns:
+        bool: True on success, False otherwise
+    """
+    return switch_to_project(defaults.ROOK_CLUSTER_NAMESPACE)
```

Comment:
@jilju by default project, user may assume the OCP namespace called "default".

Hence, either change the function name to "switch_to_default_rook_cluster_project" or some other name.

Reply:
Done